runtime-tests/endpoints/polkadotEndpoints.ts

```diff
@@ -0,0 +1,12 @@
+export const polkadotEndpoints = [
+	['/blocks/943438', '943438'], //v17
```

Comment:
Is it redundant to have the same number as next to the URL?

Reply:
So this is actually to stop us from having to parse the endpoint for the block number. I use the block number to confirm the request was successful. Each of the values in each array are passed as arguments into the callback that `test.each` returns.